openmetadata-docs/content/deployment/bare-metal/index.md

```diff
@@ -39,6 +39,17 @@ You can refer a sample script [here](https://github.com/open-metadata/OpenMetada
 
 </Note>
 
+## Postgres (version 0.12.0 or greater)
```

Comment:
For postgres we don't yet support version 15. So version should be from 12 to 14.

15 gave us some weird issues, we'll need to look into that

Reply:
```suggestion
## Postgres (version between 12.0 and 14.6)
```